capstone-app/src/app/modules/maps/map/map.component.html

```diff
@@ -8,22 +8,22 @@
       <mat-icon>close</mat-icon>
     </button>
 </mat-form-field>
-<button mat-icon-button color="primary" (click)="searchButton(searchBar.value)" >
-    <mat-icon >search</mat-icon>
+<button mat-icon-button color="primary" (click)="ngAfterViewInit()" >
```

Comment:
We shouldn't directly call a lifecycle method from the template. Instead, make a new method that the template can call, and ngAfterViewInit can call that same method from within it.